data/strings/sound.txt

```diff
@@ -409,7 +482,7 @@
     hu = Hajtson ki!
     id = Keluar.
     it = Uscita.
-    ja = 出口.
+    ja = 出口です
```

Comment:
I also see that other translations also missing a dot. I remember that the TTS engine changes pronunciation depending on the dot presence.

Reply:
It'd be interesting to hear if that was the case, I haven't noticed it in my tests.